cmd/scheduled-feed/main.go

```diff
@@ -24,11 +24,11 @@ type FeedHandler struct {
 
 func (handler *FeedHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	cutoff := time.Now().UTC().Add(-delta)
-	pkgs, err := handler.scheduler.Poll(cutoff)
-	if err != nil {
-		log.Errorf("error polling for new packages: %v", err)
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
+	pkgs, errs := handler.scheduler.Poll(cutoff)
+	if len(errs) > 0 {
+		for _, err := range errs {
+			log.Errorf("error polling for new packages: %v", err)
```

Comment:
Why not return http.Error?

Reply:
I could adjust things further down to return an http error if any feed returned an error, but I wanted to hold off from short circuiting completely here since that would mean that packages we did get wouldn't get sent upstream.

I think it's worth the extra little bit of code at the bottom so I'll add that and merge.

Reply:
Cool, sounds good. Thanks